### PR TITLE
Financial page layout

### PR DESCRIPTION
**Pull request by iandenty**

### CODE REVIEW

**Comment by iandenty** [comment]
@andresgnlez images and initial layout amended, would be discuss:

-  changing order of grid items across break points
- vertical middle grid item content

File: app/views/data_portal/index.html.erb [hunk 1]
```diff
@@ -56,11 +56,27 @@
...
+                :class => 'datasetimage -right')
+              %>
+            </div>
+          </div>
         </div>
       </section>
       <section class="l-dataset -financial-diaries" id="financial-diaries-dataset">
         <div class="row">
           <div class="grid-s-12 grid-m-6">
+            <div class="datasetimagewrapper">
```
Comment (andresgnlez) [inline app/views/data_portal/index.html.erb]: I've noticed you use this kind of nomenclature for CSS, the one we use is SMACSS (https://smacss.com/), so this class would be `dataset-image-wrapper`.

**Comment by andresgnlez** [comment]
**changing the order of grid items across breakpoints**:
Using a flex container you should be able to achieve this: when the content is a `row` you can play with `flex-direction: row | row-reverse` and when it's a `column` with `flex-direction: column | column-reverse`, does this work for you?

**vertical middle grid item content**:
Again a flex container, taking the height of the container who contains the container itself, should do it. The problem you can find here is you have in the middle some grid element that prevents to take the height, then you will have to do some "trick", modifying this grid element just there. (open to suggestions here)

File: app/views/data_portal/index.html.erb (hunk 1 above)
Comment (iandenty) [reply app/views/data_portal/index.html.erb]: was following the rcss convention for components/elements that you sent the other week

**Pull request merged**